### ISSUE TITLE
A Tip

### ISSUE THREAD

**Issue by keyhan** [kpavlov/jreactive-8583#8]
I have just created a repo based on j8583 but with some added features, I think it could be useful for your project. Look it up [jBSBE](https://github.com/keyhan/jBSBE)

**Comment by kpavlov** [comment]
Thanks. Am I right that providing I50Factory to the server or client does the job?
```java
I50Factory messageFactory = new I50Factory(SimpleTransformer.class);
Iso8583Client<I50Message> client = new Iso8583Client<>(messageFactory);
Iso8583Server<I50Message> server = new Iso8583Server<>(port, messageFactory);
```

**Comment by keyhan** [comment]
Yes, It is actually subclass to MessageFactory<I50Message> and I50Message is of course subclass to IsoMessage, so it will have the exact same behavior. What is difference are the pointe I mention in the page. The best thing I think is that if you have already a pojo with all the right values then you can send it in a REST body and the server will just transform it into a an ISO Message and send it.

**Comment by keyhan** [comment]
I have also added new annotation "@AutoStan" @kpavlov
